src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,40 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
-def merge( arrA, arrB ):
-    elements = len( arrA ) + len( arrB )
+# TO-DO: complete the helper function below to merge 2 sorted arrays
+def merge(arrA, arrB):
+    elements = len(arrA) + len(arrB)
     merged_arr = [0] * elements
-    # TO-DO
-    
+    j = 0
+    k = 0
+    for i in range(0, len(merged_arr)):
+        if j == len(arrA) and k < len(arrB):
```

Comment:
It might be a good idea to throw in some comments like "if arrA is completed..."